hms-app/.eslintrc.json

```diff
@@ -24,11 +24,17 @@
         "prettier"
     ],
     "rules": {
+        "indent": ["error", 2],
```

Comment:
no `comma-dangle` in this one

Reply:
thats correct

Reply:
thats also correct

Reply:
Jokes aside 
in the frontend setting of prettier its set to ""trailingComma": "es5","
but could not find a fitting setting für comma-dangle so I leave it out as I assume everyone in frontend will use prettier